packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -697,13 +697,34 @@ export function isParentOf(element, targetElement) {
  * @param {string} text
  */
 export async function copyToClipboard(text) {
-  await navigator.clipboard.writeText(text);
+  if (navigator?.clipboard?.writeText) {
+    try {
+      await navigator.clipboard.writeText(text);
+    } catch (err) {
+      copyToClipboardFallback(text);
+    }
+  } else {
+    copyToClipboardFallback(text);
+  }
```

Comment:
Can't you just do? Would the catch, catch the not-available navigator api if it's not present?

```suggestion
try {
    await navigator.clipboard.writeText(text);
} catch (err) {
    copyToClipboardFallback(text);
}
```

Reply:
yes better!